components/widgets/users/avatar.tsx

```diff
@@ -17,7 +23,7 @@ const Avatar = ({url, username, size}) => (
 Avatar.propTypes = {
```

Comment:
We no longer need the prop types here, is generated automatically from the types.